independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/ObserverInfo.java

```diff
@@ -75,16 +88,18 @@ static ObserverInfo create(BeanInfo declaringBean, MethodInfo observerMethod, In
 
         if (!transactionPhase.equals(TransactionPhase.IN_PROGRESS) && !jtaCapabilities) {
             final ClassInfo clazz = observerMethod.declaringClass();
-            LOGGER.warnf("The method %s#%s makes use of '%s' transactional observers but no " +
+            LOGGER.warnf("The observer %s#%s makes use of '%s' transactional observers but no " +
```

Comment:
`observer method`? ;-)

Reply:
Good catch! This could even fail with NPE!